cmd/shortener/main.go

```diff
@@ -1,3 +1,42 @@
 package main
 
-func main() {}
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/llravell/go-shortener/internal/api"
+	"github.com/llravell/go-shortener/internal/config"
+	"github.com/llravell/go-shortener/internal/models"
+	"github.com/llravell/go-shortener/internal/storages"
+)
+
+func buildConfig() (*config.Config, error) {
+	c := config.WithDefaultValues()
+	p := config.MakeParseStrategy(
+		config.NewArgsParser(c),
+		config.NewEnvParser(c),
+	)
+
+	err := p.Parse()
+	return c, err
+}
+
+func runServer(c *config.Config) error {
```

Comment:
[SUGGESTION] If I were you, I would move it to the `app` package.
https://github.com/golang-standards/project-layout/tree/master/internal/app/_your_app_